closed/custom/modules/java.base/Copy.gmk

```diff
@@ -30,6 +30,28 @@ $(call openj9_copy_files,, \
 	$(OPENJ9_VM_BUILD_DIR)/include/jvmti.h \
 	$(INCLUDE_TARGET_DIR)/jvmti.h)
 
+ifeq ($(OPENJDK_TARGET_OS), zos)
```

Comment:
nit: for consistency, the arguments should be reversed (see [OpenJ9.gmk](https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/openj9/closed/OpenJ9.gmk#L94)):
```
ifeq (zos,$(OPENJDK_TARGET_OS))
```
and the endif line should just be:
```
endif # zos
```
Same goes for the conditional block which starts on line 102.

Reply:
Updated under 48c8f92. Thanks!